ui_framework/components/modal/modal.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classnames from 'classnames';
+
+export function KuiModal({ className, children, ...rest }) {
+  const classes = classnames('kuiModal', className);
+  return (
+    <div className={ classes } { ...rest }>
+      { children }
+    </div>
+  );
+}
+
+KuiModal.propTypes = {
+  className: React.PropTypes.string,
```

Comment:
More `React.PropTypes`, which we should stop using. Please use the `prop-types` module.